src/components/TileGrid/InfoTile.tsx

```diff
@@ -34,7 +34,7 @@ const InfoTile = (params: InfoTileProps) => {
 
   return (
     <div className="flex h-fit w-full flex-col gap-y-4 rounded-lg bg-white p-6 shadow-lg">
-      <div className="flex justify-between">
+      <div className="relative flex justify-between">
```

Comment:
We need this for the popup with the `...` icon. Otherwise, it will fly out of the screen.